.github/workflows/linux.yml

```diff
@@ -108,3 +113,59 @@ jobs:
         if: matrix.cli == true
         run: sh validate.sh -j 2 -w ghc-${{ matrix.ghc }} -v ${{ matrix.flags }} -s cli-suite
 
+  validate-old-ghcs:
+    name: Validate old ghcs ${{ matrix.extra-ghc }}
+    runs-on: ubuntu-18.04
+    needs: validate
```

Comment:
Can you leave a comment here to clarify that this `needs` is only to limit concurrency? Otherwise I would expect this job to depends on assets produced earlier.

Reply:
I think limit concurrency is not the intent from the original `needs`, which I somewhat replicated here
The idea would be you have to ensure the main ghc is validated testing itself before using it to validate old ghcs, to not mix possible errors
It should only depend on the Linux ghc-8
8.4 from the previous job and no the entire one but I did not get to accomplish it

Reply:
> It should only depend on the Linux ghc-8.8.4 

It seems you cant depend on a specific matrix job, the job id for all of them is the same (`validate`)